pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/logging/LogUtil.java

```diff
@@ -15,4 +15,8 @@ public static void addAppEngineEnvVarsToMDC() {
         MDC.put(GAE_DEPLOYMENT_ID, System.getProperty(GAE_DEPLOYMENT_ID));
         MDC.put(GAE_INSTANCE, System.getProperty(GAE_INSTANCE));
     }
+
```

Comment:
I have no idea what this class does.
Would JavaDoc for the class help?

Reply:
Done, thank you for flagging!